Libraries/pbxbuild/Headers/pbxbuild/Tool/CopyResolver.h

```diff
@@ -34,13 +34,6 @@ class CopyResolver {
         std::string const &outputDirectory,
```

Comment:
Always pass in inputs as tool inputs, rather than path strings: LGTM since I commented with nits.